### ISSUE TITLE
Switched from 0.9.1 to 0.10.1. Settings in the pom.xml in generationConfig have no effect anymore.

### ISSUE THREAD

**Issue by mzzc** [phoenixnap/springmvc-raml-plugin#133]
Switched from 0.9.1 to 0.10.1. Settings in the pom.xml in generationConfig have no effect anymore.

For instance here is a snippet from my pom. The only setting that works is useCommonsLang3. Especially settings like includeHashcodeAndEquals and includeToString are important.

```
<generationConfig>
    <includeAdditionalProperties>true</includeAdditionalProperties>
    <includeHashcodeAndEquals>false</includeHashcodeAndEquals>
    <includeToString>false</includeToString>
    <useLongIntegers>true</useLongIntegers>
    <usePrimitives>true</usePrimitives>
    <generateBuilders>true</generateBuilders>
    <useCommonsLang3>true</useCommonsLang3>
    <targetVersion>1.8</targetVersion>
</generationConfig>
```

Also the generated serialVersionUID is of Type Long instead long. The compiler marks that as a warning.

**Comment by kurtpa** [comment]
If you are using RAML 1.0, i'm afraid that the generation config is no longer the same. Currently this is the implemented subset: https://github.com/phoenixnap/springmvc-raml-plugin/blob/master/springmvc-raml-parser/src/main/java/com/phoenixnap/oss/ramlapisync/pojo/PojoGenerationConfig.java

I will check regarding serialVersionUID :)

**Comment by mzzc** [comment]
Now I tried only this:

```
<generationConfig>						
    <generateHashcodeEqualsToString>false</generateHashcodeEqualsToString>
</generationConfig>
```

and

```
<generationConfig>						
    <isGenerateHashcodeEqualsToString>false</isGenerateHashcodeEqualsToString >
</generationConfig>
```

and

```
<generationConfig>						
    <withHashcodeEqualsToString>false</withHashcodeEqualsToString >
</generationConfig>
```

I get following mvn compile error:
`[ERROR] Failed to execute goal com.phoenixnap.oss:springmvc-raml-plugin:0.10.1:generate-springmvc-endpoints (generate-springmvc-endpoints) on project customer-promise-and-fulfillment-sourcing: Unable to parse configuration of mojo com.phoenixnap.oss:springmvc-raml-plugin:0.10.1:generate-springmvc-endpoints for parameter generateHashcodeEqualsToString: Cannot find 'generateHashcodeEqualsToString' in class com.phoenixnap.oss.ramlapisync.plugin.JsonShema2PojoGenerationConfig -> [Help 1]`

What is the correct naming of the settings? Could you give me an example?

**Comment by kurtparis** [comment]
https://github.com/phoenixnap/springmvc-raml-plugin/blob/master/springmvc-raml-plugin/src/main/java/com/phoenixnap/oss/ramlapisync/plugin/SpringMvcEndpointGeneratorMojo.java contains the code to map the config. Upon reviewing it i noticed that the hashcode and equals config is missing. I'll commit a fix for inclusion in the upcoming 0.10.2. Sorry!

**Comment by kurtpa** [comment]
Added a commit which will link the hashcode and equals using the jsonSchema2Pojo config as before. Please note that you can only switch on or off all of hashCode,equals,toString or none of them. ie:

<generationConfig>						
    <includeToString>false</includeToString>
    <includeHashcodeAndEquals>false</includeHashcodeAndEquals>
</generationConfig>
